unlock-wordpress-plugin/inc/classes/fullpostpage/class-editfullpp.js

```diff
@@ -0,0 +1,323 @@
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
Is this standard, I am surprised you would need to do that...

Reply:
We should use Gutember blocks for all this. Please check how the plugin does it for the "blocks".

Reply:
fixed by using reacjs in the "src" folder